supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -133,9 +133,9 @@ class JesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
   }
 
   /**
-    * Takes two arrays of remote and local WDL File paths and generates the necessary JesInputs.
+    * Takes two arrays of remote and local WOM File paths and generates the necessary JesInputs.
     */
-  private def jesInputsFromWdlFiles(jesNamePrefix: String,
+  private def jesInputsFromWomFiles(jesNamePrefix: String,
```

Comment:
Might as well rename fully to `papiInputsFromWomFiles`?

Reply:
At some point we should PAPI All The Things but I think that's a separate PR. This is just the stuff I bumped into fixing `writeFile`.